docs/collection_metadata.rst

```diff
@@ -6,21 +6,42 @@ will contribute their Python and system requirements to the image.
 
 Requirements from a collection can be recognized in these ways:
 
--  A file ``meta/execution-environment.yml`` references the Python
-   and/or bindep requirements files
--  A file named ``requirements.txt`` is in the root level of the
-   collection
--  A file named ``bindep.txt`` is in the root level of the collection
+* The ``meta/execution-environment.yml`` file containing the Python
+  and/or bindep requirements or referencing other files listing them.
+* The ``requirements.txt`` file in the root level of the collection.
+* The ``bindep.txt`` file in the root level of the collection.
 
 If any of these files are in the ``build_ignore`` of the collection, it
 will not work correctly.
 
+How to verify collection-level metadata
+=======================================
+
 Collection maintainers can verify that ``ansible-builder`` recognizes
```

Comment:
The only thing I might bring attention to is that this is an optional step, useful for troubleshooting if needed and not part of the typical workflow for building an exection environment.

Reply:
Seems like most of this page is not directed to those building EEs, but to collection maintainers to ensure their collections work in EEs? If so, maybe append 'for collection maintainers' to the page title.

Reply:
Added a note, PTAL